README.md

```diff
@@ -52,11 +56,28 @@ lock:
       name: "Nuki Door Sensor"
     door_sensor_state:
       name: "Nuki Door Sensor State"
+    unpair:
+      name: "Nuki Unpair"
+    pairing_mode:
+      name: "Nuki Pairing Mode"
+
+  # Optional: Settings
+    security_pin: 1234
+    pairing_mode_timeout: 300s
```

Comment:
Did you mean `pairing_timeout`?

Reply:
Oh yea you're right. Would you prefer the naming pairing_mode_timeout or pairing_timeout because it's related to the pairing mode but not to the pairing itself.

I just thought it was pairing_mode_timeout because it's more reasonable and didn't look at the code when I changed it in the readme 😅

Reply:
Sure, pairing_mode_timeout sounds more reasonable.

Reply:
Okay good, I'm gonna change that tomorrow when I'm back home :)

Reply:
I changed it and found another small mistake when not rendering the template properly

Reply:
Great, thanks.